src/MultiSelect/MultiSelect.svelte

```diff
@@ -156,6 +156,12 @@
    */
   export let selectionRef = null;
 
+  /**
+   * Id of the highlighted ListBoxMenuItem
+   * @type {null | string}
```

Comment:
We should reuse the `MultiSelectItemId` typedef.

```suggestion
   * @type {null | MultiSelectItemId}
```

Reply:
done